classes/lang/KeysReference/MetaLang.php

```diff
@@ -56,26 +56,26 @@
 
 trans('Shop powered by PrestaShop', 'Shop.Navigation');
 
-trans('Brand list', 'Shop.Navigation');
```

Comment:
Maybe we could use the existing string "List of all brands" in Shop.Theme.Global to avoid creating a new string.